generate_grid.py fails with FileNotFoundError for tilegrid.json

```
074-dump_all             - 24m29s:                                                                                
074-dump_all             - 24m29s: 
074-dump_all             - 24m29s: 100% (1953452 of 1953452) |##############| Elapsed Time: 0:04:15 Time:  0:04:15
074-dump_all             - 24m29s: + python3 reduce_site_types.py --output_dir build/output
074-dump_all             - 24m29s: + python3 generate_grid.py --root_dir build/specimen_001/ --output_dir build/output --ignored_wires artix7_ignored_wires.txt
074-dump_all             - 24m29s: Traceback (most recent call last):
074-dump_all             - 24m29s:   File "generate_grid.py", line 676, in <module>
074-dump_all             - 24m29s:     main()
074-dump_all             - 24m29s:   File "generate_grid.py", line 578, in main
074-dump_all             - 24m29s:     with open(os.path.join(prjxray.util.get_db_root(), 'tilegrid.json')) as f:
074-dump_all             - 24m29s: FileNotFoundError: [Errno 2] No such file or directory: '/tmpfs/src/github/symbiflow-prjxray-continuous-db-artix7/database/artix7/tilegrid.json'
074-dump_all             - 24m29s: make[2]: *** [build/specimen_001/OK] Error 1
074-dump_all             - 24m29s: make[1]: *** [run] Error 2
074-dump_all             - 24m29s: 
074-dump_all             - 24m29s: --------------------------------------------------------------------------
074-dump_all             - 24m29s: Failed @ 2019-02-03T08:38:34.923364 with exit code: 2
```

--- a/fuzzers/Makefile
+++ b/fuzzers/Makefile
@@ -54,7 +54,7 @@ $(eval $(call fuzzer,071-ppips,057-pip-bi 058-pip-hclk 060-bram-cascades))
 ifneq ($(BITONLY),Y)
 $(eval $(call fuzzer,072-ordered_wires,))
 $(eval $(call fuzzer,073-get_counts,))
-$(eval $(call fuzzer,074-dump_all,072-ordered_wires))
+$(eval $(call fuzzer,074-dump_all,005-tilegrid 072-ordered_wires))
 endif
 endif
 $(eval $(call fuzzer,100-dsp-mskpat,005-tilegrid))

